edagames/settings/prod.py

```diff
@@ -89,15 +91,47 @@
 WSGI_APPLICATION = 'edagames.wsgi.application'
 
 
+# get secret
+def get_secret(db_secret_name):
+
+    secret_name = db_secret_name
+    region_name = "us-east-1"
+
+    # Create a Secrets Manager client
+    session = boto3.session.Session()
+    client = session.client(
+        service_name='secretsmanager',
+        region_name=region_name
+    )
+    try:
+        get_secret_value_response = client.get_secret_value(
+            SecretId=secret_name
+        )
+    except ClientError as e:
+        raise e
+    else:
+        # Decrypts secret using the associated KMS key.
+        # Depending on whether the secret is a string or binary, one of these fields will be populated.
+        if 'SecretString' in get_secret_value_response:
+            secret_value_response = get_secret_value_response['SecretString']
+        else:
+            secret_value_response = base64.b64decode(get_secret_value_response['SecretBinary'])
```

Comment:
This might be a unnecessary else block right? I mean

```
try:
    ...
except ClientError as e:
    ...

if 'SecretString' ...:
    ...
else:  
    ...
```

Reply:
Yes, It looks better that way. Thanks!